CHANGELOG.md

```diff
@@ -101,38 +104,30 @@ experiment with it, but do not ship production code using it yet.
 
 ### Spanner
 
-* feat(spanner): use multiple channels instead of multiple client for the multiple rows cpu benchmark (#6659)
+* feat(spanner): use multiple channels instead of multiple client for the
+  multiple rows cpu benchmark (#6659)
 * fix(spanner): do not warn about SessionPoolClockOption (#6619)
 
 ### Storage
 
 * feat(storage): recommend `g::c::Options` (#6640)
-* feat: custom trust store with unified credentials (#6617)
-* feat(storage): add Retry Test API to emulator (#6611)
-* fix(storage): address behavior bugs (#6574)
 * feat(storage): support multiple channels in GCS+gRPC (#6593)
 * feat(storage): round-robin stub for GCS+gRPC (#6584)
 * fix(storage): compile GCS+gRPC with Windows+x86 (#6556)
-* feat: implement unified service account credentials (#6531)
 * feat(storage): install rules for GCS+gRPC plugin (#6527)
-* feat: implement unified insecure credentials (#6518)
 * fix(storage): compile with GCC 11 in C++17 mode (#6501)
-* feat(storage): support service account impersonation (#6488)
-* feat(storage): support method overrides in emulator (#6474)
-* fix(storage): support gzip request body in emulator (#6472)
-* fix(storage): update emulator to use MultipartDecoder (#6453)
-* feat(storage): impersonate service account credentials (#6441)
+* feat: Support unified credentials (`google::cloud::Credentials`) to initialize
+  both the REST and gRPC plugins with the same classes. (#6518)(#6531)(#6617)(#6488)(#6441)
 
 ### Common Libraries
 
-* feat: custom trust store with unified credentials (#6617)
-* feat(common): make unified Credentials public (#6614)
+* feat: Add support for "unified" credentials, which allow applications to
+  Configure the REST and gRPC-based libraries using the same credential classes
+  Support complex credential types, such as service account impersonation
+  With this release, only usable in the `storage` library supports these classes.
```

Comment:
Ugh, sorry, my grammar was awful here .. maybe "With this release only the `storage` library supports unified credentials"?